charts/jaeger/templates/agent-ds.yaml

```diff
@@ -84,6 +84,30 @@ spec:
             port: admin
         resources:
 {{ toYaml .Values.agent.resources | indent 10 }}
+        volumeMounts:
+        {{- range .Values.agent.extraConfigmapMounts }}
+          - name: {{ .name }}
+            mountPath: {{ .mountPath }}
+            subPath: {{ .subPath }}
+            readOnly: {{ .readOnly }}
+        {{- end }}
+        {{- range .Values.agent.secretMounts }}
```

Comment:
nit: Can we call these `extraSecretMounts` please? Just to match naming of `extraConfigmapMounts`

Reply:
I'm closing this PR. I will create PRs one by one describing purpose of each